official/wide_deep/census_main.py

```diff
@@ -111,6 +111,6 @@ def main(_):
 
 
 if __name__ == '__main__':
-  tf.logging.set_verbosity(tf.logging.INFO)
+  tf.compat.v1.logging.set_verbosity(tf.compat.v1.logging.INFO)
```

Comment:
There must be a V2 way to do logging not with tf.compat.v1.  The other person reviewing should know the right call.  If not, oh well.